components/serverless/internal/controllers/serverless/utils.go

```diff
@@ -186,22 +186,48 @@ func mapsEqual(existing, expected map[string]string) bool {
 	return true
 }
 
+func mapsContains(existing, contains map[string]string) bool {
```

Comment:
I have problem with understanding which value is which - `existing` and `contains` say me nothing.